aiogram/dispatcher/filters/builtin.py

```diff
@@ -691,3 +691,17 @@ def __init__(self, is_forwarded: bool):
 
     async def check(self, message: Message):
         return bool(getattr(message, "forward_date")) is self.is_forwarded
+
+
+class ChatTypeFilter(BoundFilter):
+    key = 'chat_types'
+
+    def __init__(self, chat_types: typing.List[ChatType]):
+        self.chat_types = chat_types
```

Comment:
Can you add support for single `ChatType`?

For example:
```python3
@dp.message_handler(chat_types=ChatType.PRIVATE)
```

Reference: `ContentTypeFilter` https://github.com/aiogram/aiogram/blob/006a5ef987b4bc59feb78b9ff3f54061f273f4b0/aiogram/dispatcher/filters/builtin.py#L459

Reply:
In order to avoid confusion which we have with TextFilter, I decided not to add it. Instead, I think we should force users to use lists as input arguments in such filters

Reply:
implemented at https://github.com/aiogram/aiogram/pull/356/commits/768491d97604e9e25d53f432fe12c597121ee88e

Reply:
@JrooTJunior, your thoughts? There has to be some kind of unified approach